homeassistant/components/tuya/translations/es.json

```diff
@@ -40,7 +40,8 @@
                     "support_color": "Forzar soporte de color",
                     "temp_divider": "Divisor de los valores de temperatura (0 = usar valor por defecto)",
                     "tuya_max_coltemp": "Temperatura de color m\u00e1xima notificada por dispositivo",
-                    "unit_of_measurement": "Unidad de temperatura utilizada por el dispositivo"
+                    "unit_of_measurement": "Unidad de temperatura utilizada por el dispositivo",
+                    "precision_override": "Precisi\u00f3n de temperatura"
```

Comment:
Do not provide additional translations. Just adjusting string.json is enough, the rest is handled by the translation platform.

Please remove all additional translations.

Reply:
Ops, it was easier than I thought.
Added in string.json and removed from translations.

Reply:
Since I removed all the translations and I kept only strings.json, I don't see any labels during testing in my local environment. Is that ok? How can assure that it will work?
I'm just worried.
If it's OK, please resolve this thread.

Thank you for your time.

Reply:
@ollo69 please help me here, too.